gui/wxpython/startup/guiutils.py

```diff
@@ -218,7 +218,7 @@ def create_location_interactively(guiparent, grassdb):
     """
     from location_wizard.wizard import LocationWizard
 
-    gWizard_output = (None, None, None)
+    gWizard_output = (None, None, "PERNAMENT")
```

Comment:
The mapset name here is little strange as a value on failure. The original `(None, None, None)` sounds better and is probably safer.

Reply:
Changed.